Bio/GenBank/Scanner.py

```diff
@@ -476,16 +477,16 @@ def parse_records(self, handle, do_features=True):
         """
         # This is a generator function
         while True:
-            record = self.parse(handle, do_features)
-            if record is None:
+            next_record = self.parse(handle, do_features)
```

Comment:
`record` shadows an outer scope variable. I'd rather use the shorter version as well, but it's safer to use different variable names IMO. How about changing `next_record` to `rec`?